src/mixxxapplication.cpp

```diff
@@ -181,10 +182,15 @@ bool MixxxApplication::notify(QObject* target, QEvent* event) {
 #endif
 
     PerformanceTimer time;
-    time.start();
+    bool isDeveloper = CmdlineArgs::Instance().getDeveloper();
```

Comment:
I'm curious: isquerying a bool for each event as fast as using a (const) bool member for this constant property?

Reply:
I cache this in a member variable now, which is initialized in the constructor.

Reply:
> I'm curious: isquerying a bool for each event as fast as using a (const) bool member for this constant property?

If the entire thing is completely inlined, maybe. but caching makes sense here.